FCCee/CLD/compact/CLD_o2_v05/CLD_o2_v05.xml

```diff
@@ -0,0 +1,433 @@
+<lccdd xmlns:compact="http://www.lcsim.org/schemas/compact/1.0"
+    xmlns:xs="http://www.w3.org/2001/XMLSchema"
+    xs:noNamespaceSchemaLocation="http://www.lcsim.org/schemas/compact/1.0/compact.xsd">
+
+    <info name="CLD_o2_v05"
+        title="FCCee detector model option 2 version 05 (updated Beampipe and VXD)"
+	author="Andre Sailer"
+        url="http://ilcsoft.desy.de"
+        status="development"
+        version="5">
+        <comment>The compact format for the FCCee Detector design</comment>
+    </info>
+
+    <includes>
+        <gdmlFile  ref="elements.xml"/>
+        <gdmlFile  ref="materials.xml"/>
+    </includes>
+
+    <define>
+        <constant name="world_side" value="6100*mm"/>
+        <constant name="world_x" value="world_side"/>
+        <constant name="world_y" value="world_side"/>
+        <constant name="world_z" value="world_side"/>
+
+        <constant name="CrossingAngle" value="0.030*rad"/>
+
+        <constant name="SolenoidField" value="2*tesla"/>
+
+        <constant name="DetID_NOTUSED"          value=" 0"/>
+
+        <constant name="DetID_VXD_Barrel"       value=" 1"/>
+        <constant name="DetID_VXD_Endcap"       value=" 2"/>
+
+        <constant name="DetID_IT_Barrel"        value=" 3"/>
+        <constant name="DetID_IT_Endcap"        value=" 4"/>
+
+        <constant name="DetID_OT_Barrel"        value=" 5"/>
+        <constant name="DetID_OT_Endcap"        value=" 6"/>
+
+        <constant name="DetID_ECal_Barrel"      value=" 20"/>
+        <constant name="DetID_ECal_Endcap"      value=" 29"/>
+        <constant name="DetID_ECal_Plug"        value=" 21"/>
+
+        <constant name="DetID_HCAL_Barrel"      value=" 10"/>
+        <constant name="DetID_HCAL_Endcap"      value=" 11"/>
+        <constant name="DetID_HCAL_Ring"        value=" 12"/>
+
+        <constant name="DetID_Yoke_Barrel"      value=" 13"/>
+        <constant name="DetID_Yoke_Endcap"      value=" 14"/>
+
+        <constant name="DetID_LumiCal"          value=" 15"/>
+        <constant name="DetID_LumiCalInstrumentation"          value=" 16"/>
+        <constant name="DetID_LumiCalBackShield"          value=" 18"/>
+        <constant name="DetID_LumiCalCooling"          value=" 17"/>
+        <constant name="DetID_LumiCalNoseShield"          value=" 22"/>
+
+        <constant name="DetID_HOMAbsorber"          value=" 19"/>
+
+        <constant name="BeamPipeWidth"   	value="1.7*mm" />
```

Comment:
For the other change proposed above in the beampipe, please also add here the following:
 ```
       <constant name="BeamPipeWidthWall"      value="0.35*mm" />
        <constant name="BeamPipeWidthCooling"   value="1.0*mm" />
        <constant name="BeamPipeWidth"   	    value="2.0*BeamPipeWidthWall + BeamPipeWidthCooling"/> <!-- 1.7 mm in total -->
```

Reply:
And if tolerances are needed as well, then this:
`        <constant name="BeamPipeTolerance"   	value="0.001*mm"/>`